appinventor/buildserver/src/com/google/appinventor/buildserver/YoungAndroidConstants.java

```diff
@@ -28,9 +28,9 @@ private YoungAndroidConstants() {}
 
   /**
    * The filename extension for the file which contains block metadata to be
-   * consumed by codeblocks.
+   * consumed by Blockly.
    */
-  public static final String CODEBLOCKS_SOURCE_EXTENSION = ".blk";
+  public static final String BLOCKS_SOURCE_EXTENSION = ".bky";
```

Comment:
Note there is also a constant in appengine of the same (original) name and value. We may want to consider adding this rather than renaming the existing one just to keep the modules consistent.